consensus/consensus-types/src/sync_info.rs

```diff
@@ -18,6 +18,13 @@ pub struct SyncInfo {
     highest_timeout_cert: Option<TimeoutCertificate>,
 }
 
+// this is required by structured log
+impl std::fmt::Debug for SyncInfo {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
```

Comment:
super nit: std::fmt::Formatter is already in scope and you could trivially make std::fmt::Debug into scope.